package.json

```diff
@@ -39,59 +39,59 @@
-    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
-    "@babel/plugin-proposal-pipeline-operator": "^7.0.0",
-    "@babel/plugin-proposal-throw-expressions": "^7.0.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
-    "@babel/plugin-syntax-import-meta": "^7.0.0",
-    "@babel/plugin-transform-runtime": "^7.1.0",
-    "@babel/preset-env": "^7.1.5",
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/plugin-proposal-class-properties": "^7.1.2",
+    "@babel/plugin-proposal-do-expressions": "^7.1.2",
+    "@babel/plugin-proposal-export-default-from": "^7.1.2",
+    "@babel/plugin-proposal-export-namespace-from": "^7.1.2",
+    "@babel/plugin-proposal-json-strings": "^7.1.2",
+    "@babel/plugin-proposal-logical-assignment-operators": "^7.1.2",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.1.2",
+    "@babel/plugin-proposal-numeric-separator": "^7.1.2",
+    "@babel/plugin-proposal-optional-chaining": "^7.1.2",
+    "@babel/plugin-proposal-pipeline-operator": "^7.1.2",
+    "@babel/plugin-proposal-throw-expressions": "^7.1.2",
+    "@babel/plugin-syntax-dynamic-import": "^7.1.2",
+    "@babel/plugin-syntax-import-meta": "^7.1.2",
+    "@babel/plugin-transform-runtime": "^7.1.2",
+    "@babel/preset-env": "^7.1.2",
     "@babel/preset-flow": "^7.0.0",
-    "@babel/runtime": "^7.1.5",
+    "@babel/runtime": "^7.1.2",
     "@integreat-app/eslint-config-babel": "^0.3.1",
     "@integreat-app/eslint-config-flow": "^0.3.2",
     "@integreat-app/eslint-config-jest": "^0.2.1",
     "@integreat-app/eslint-config-node": "^0.3.2",
-    "babel-core": "^6.26.3",
     "babel-eslint": "^10.0.1",
-    "babel-jest": "^23.6.0",
-    "babel-loader": "^8.0.4",
+    "babel-jest": "^24.0.0",
+    "babel-loader": "^8.0.5",
     "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-preset-env": "^1.7.0",
-    "es-check": "^4.0.0",
-    "eslint": "^5.9.0",
+    "es-check": "^5.0.0",
+    "eslint": "^5.12.1",
     "eslint-config-standard": "^12.0.0",
-    "eslint-plugin-flowtype": "^3.2.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jest": "^22.0.0",
-    "eslint-plugin-node": "^8.0.0",
+    "eslint-plugin-flowtype": "^3.2.1",
+    "eslint-plugin-import": "^2.15.0",
+    "eslint-plugin-jest": "^22.1.3",
+    "eslint-plugin-node": "^8.0.1",
     "eslint-plugin-prefer-object-spread": "^1.2.1",
     "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-standard": "^4.0.0",
-    "flow-bin": "^0.86.0",
+    "flow-bin": "^0.91.0",
     "flow-copy-source": "^2.0.2",
     "flow-typed": "^2.5.1",
-    "jest": "^23.6.0",
-    "jest-fetch-mock": "^1.7.4",
-    "jest-junit": "^5.2.0"
+    "jest": "^24.0.0",
+    "jest-fetch-mock": "^2.1.0",
+    "jest-junit": "^6.1.0",
+    "shelljs": "^0.8.3",
+    "webpack": "^4.29.0",
+    "webpack-cli": "^3.2.1"
   },
   "dependencies": {
     "entities": "^1.1.2",
     "is-url": "^1.2.4",
-    "lodash-es": "^4.17.11",
-    "moment": "^2.22.2",
+    "lodash": "^4.17.11",
```

Comment:
I don't know about the current output size, but if its too large, you could consider using `babel-plugin-lodash` to try to minimize it further

Reply:
For now it is oke :)